be/src/olap/iterators.h

```diff
@@ -82,6 +83,7 @@ class StorageReadOptions {
     // reader's column predicate, nullptr if not existed
     // used to fiter rows in row block
     std::vector<ColumnPredicate*> column_predicates;
+    std::vector<ColumnPredicate*> all_compound_column_predicates;
```

Comment:
a more intuitive name is better

Reply:
rename to `column_predicates_except_leafnode_of_andnode`